packages/xchain-thorchain/src/client.ts

```diff
@@ -242,7 +233,10 @@ class Client implements ThorchainClient, XChainClient {
 
       const unsignedStdTx = StdTx.fromJSON({
         msg,
-        fee: this.getDefaultStdTxFee(),
+        fee: {
+          amount: [],
+          gas: '10000000',
```

Comment:
Pls provide a  `const DEFAULT_GAS_VALUE = '10000000'` or something to use in `deposit`, `transfer` and `getDefaultFees`. Just to make sure we do use same value everywhere.